### PR TITLE
Remove apste_support_on_.* from Server Certification disk tests

### PR DESCRIPTION
**Pull request by rodwsmith**
## Description

Describe your changes here:

Removed apste_support_on_.* from Server Certification disk tests

## Resolved issues

https://warthogs.atlassian.net/browse/SERVCERT-455

## Documentation

- [ ] Automated tests are included for the changed functionality in this PR. If to be merged without tests, please elaborate why.
- [ ] Necessary documentation is provided for the changed functionality in this PR (tests are documentation, too).

## Tests

- [ ] Steps to follow for reviewer to run & manually test are provided.
- [ ] A list of what tests were run and on what platform/configuration is provided.

### CODE REVIEW

File: providers/base/units/disk/test-plan.pxu [hunk 1]
```diff
@@ -67,4 +66,3 @@ include:
     disk/fstrim_.*                             certification-status=non-blocker
     disk/disk_stress_ng_.*                     certification-status=blocker
     disk/disk_cpu_load_.*                      certification-status=blocker
-    disk/apste_support_on_.*                   certification-status=non-blocker
```
Comment (bladernr) [inline providers/base/units/disk/test-plan.pxu:70]: Please verify this is not in use by the devices team (I don't think we use any of the test plans marked as "manual"

File: providers/base/units/disk/test-plan.pxu [hunk 2]
```diff
@@ -17,7 +17,6 @@ include:
     disk/storage_device_.*                     certification-status=blocker
     benchmarks/disk/hdparm-read_.*
     benchmarks/disk/hdparm-cache-read_.*
-    disk/apste_support_on_.*
```
Comment (bladernr) [inline providers/base/units/disk/test-plan.pxu:20]: This one should not be removed, this is the automated disk test plan that Devices uses, the test is OK for them as APSTE is a feature on consumer grade devices, while not being a feature on enterprise grade devices.

**Review by bladernr — CHANGES_REQUESTED** [review]
Trying an inline comment... haven't done that on Git yet...
tl;dr - the removal from disk-cert-automated should be reverted, while keeping the removal from server-disk

**Review by bladernr — CHANGES_REQUESTED** [review]
Sorry to do this, but could you squash these two commits into a single one? rather than logging two commits where the second just reverses something in the first one, it's cleaner to just squash and push a single commit.  It's pedantic, I know, but keeps the history a bit cleaner.

**Comment by rodwsmith** [comment]
> Sorry to do this, but could you squash these two commits into a single one? rather than logging two commits where the second just reverses something in the first one, it's cleaner to just squash and push a single commit. It's pedantic, I know, but keeps the history a bit cleaner.

Done.

**Review by bladernr — APPROVED** [review]

**Review by bladernr — CHANGES_REQUESTED** [review]
OK, one more thing I missed... sorry, also part of this is me learning how to review things on Github.  But you're missing an underscore where you restored this in the devices test plan.  That'll cause their tests to potentially break since the name is slightly different than expected.

**Comment by rodwsmith** [comment]
> 

This time git blew up in my face when I tried to squash my changes, so I'm closing this PR and have opened a new one: https://github.com/canonical/checkbox/pull/319
